### DISCUSSION TITLE
Welcome to TeXtured Discussions!

### DISCUSSION THREAD

**Discussion by jdujava** [jdujava/TeXtured#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by junxng** [comment]
I still don't fully understand how to properly cite articles and print them out accurately. Could you please provide me with detailed instructions? Thank you in advance.

**Reply by jdujava** [reply]
`TeXtured` uses `biblatex` package for bibliography/citation of references. Just add references with to `preamble/bibliography.bib` file, and then cite using `\cite{...}` or `\autocite{...}`. You can check how is it done in `Demo` document. For more details consult for example documentation at https://ctan.org/pkg/biblatex, or you can find plenty of guides online.

**Comment by RLukas2** [comment]
Hi, I'm interested in `TeXtured` as it looks great and is well-documented. I'd like to ask whether the current template only supports the thesis format, or if one could write code to support other formats such as reports and journals. And will there any support for these formats in the future? Thank you and wish you have a good day.

**Reply by jdujava** [reply]
Currently, `TeXtured` is designed mainly for the thesis format. It builds upon the base `report` class, that is it assumes the document hierarchy to be chapter, section, subsection, ... Thus, for any document format of this type (for example a longer report containing multiple chapters), `TeXtured` works with minimal changes (create an appropriate Title Page, include/omit certain front matter pages).

Smaller documents containing sections at the highest level (such as journals) would require additional changes. Just switching to `article` class isn't enough, since that would result in errors (due to tweaks performed by `TeXtured` assuming `report` class). Also some design/functionality should be adapted (spacing in ToC, generation of correct levels in PDF Outline,...).

I think it would be interesting to support article-type documents out of the box, but I don't plan to implement it in the nearest future. PRs are welcome, though.
